src/EventHandlerService.js

```diff
@@ -0,0 +1,81 @@
+import _ from 'lodash';
```

Comment:
perhaps instead of importing the whole library, we can bring in only what we need.
ex.: https://www.npmjs.com/package/lodash.set

Reply:
I agree, I was using it in the assignment of a deeply nested variable [here](https://github.com/makerdao/testchain-client/blob/56640fa588ce7ffb351acc13562b7d426dab4232/src/EventHandlerService.js#L42). 
I suppose a question I would have is how best is it to handle deeply nested variables in objects or is there an optimal convention to avoid the general clunkiness and eventual undefined errors of using `something1.something2.something3`.
I have found this [blog post](https://blog.osteele.com/2007/12/cheap-monads/) to help with determining the existence of deeply nested values but I would find it helpful to hear yere opinions on this? @b-pmcg @levity

Reply:
IMO using `_.set` is a ok approach, I'd opt with the array notation over strings though.

One day we might have https://github.com/tc39/proposal-optional-chaining so we'll be able to do `something1.something2?.something3` like coffeescript's existential operation, at least for getters, until then we're on our own and some people can dislike monads.

Main concern here is not bloating the project by importing big libraries like lodash all at once :)

Reply:
Sure sure. Cheers, the `?` operator would be a nice addition. I'll be sure to remember with lodash

Reply:
+1 to the suggestion of bringing lodash in piece by piece. actually i don't mind either way, because i usually end up using a dozen lodash methods and it's [not huge](https://bundlephobia.com/result?p=lodash@4.17.11), but even if you do use the whole thing, you should import only the methods you use in a given file, e.g.:
```js
import { set } from 'lodash';
```

using the `_` is a hold-over from ancient browser environments.